src/styles/_catalog.scss

```
@@ -1,22 +1,27 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  grid-auto-rows: 300px;
+  justify-content: center;
+  gap: 48px;
```

```suggestion
  gap: 48px;
```

Особисто я виніс би такі величини як відступи та ширину в змінні, щоб було легше управляти ними і за потреби робити якісь обчислення

---

my opinion - gap should be clearly defined , variables are not required. 
We are able simply change  the gap dimension directly.
Changes made with anticipation of future - is not the best practice.